apps/condo/domains/property/utils/testSchema/index.js

```diff
@@ -58,10 +59,17 @@ async function makeClientWithProperty (includeFlat = false) {
     return client
 }
 
-async function makeClientWithResidentUserAndProperty () {
-    const userClient = await makeClientWithProperty()
-    await addResidentAccess(userClient.user)
-    return userClient
+async function makeClientWithResidentAccessAndProperty () {
+    const clientWithOrganization = await makeClientWithProperty()
+    const client = await makeClientWithResidentUser()
+    const organization = clientWithOrganization.organization
+    const property = clientWithOrganization.property
+
+    await addResidentAccess(client.user)
+    client.organization = organization
+    client.property = property
```

Comment:
I think you'd also need to create the Resident model, which would be connected to this organization and this property

Reply:
There will be a lot of tests to change, I added TODO in `b66c214`